2019/NumPy_ideas_list.md

```diff
@@ -22,23 +67,66 @@ Welcome, and thank you for taking an interest in NumPy! On this page we will fir
+NumPy serves many kinds of users: students new to programming or Python,
+educators, researchers, domain experts in one of the areas that NumPy covers,
+data scientists, library developers, packagers, and more. And NumPy's
+documentation is *huge* (the pdf version of the last release is over 1500
+pages). **The challenge**: *provide ways to guide those users to the parts of
+the documentation most relevant to them.* We would love to work with a
+technical writer that is able to help us address this challenge.
 
 **Possible topics** include:
 - Improving the structure and content of https://numpy.org/
-- Reviewing and improving the structure of the main navigation page of [the NumPy documentation](https://www.numpy.org/devdocs/).
-- Producing a roadmap or list of work items for engaging the community in further documentation work (we have a lot of contributor, channeling their energy by setting the right goals and priorities could have a lot of impact). This may include making explicit what different kinds of users need to see.
-- Rewriting a section of the User Guide (that can then serve as a template for other/new sections).
-
-We assume that the technical writer who will take on this project is not yet familiar with NumPy and its community. This means time will need to be built into the project plan at the start to get familiar with it. Mentors will be able to provide walkthroughs, set up time for discussing with or interviewing different kinds of end users, or anything else that in the writers experience is a good idea. *Note that some mentors have experience working with technical writers in a day-job-in-industry setting, but not in an open source setting - so we'll all learn a lot!*
+- Reviewing and improving the structure of the main navigation page of [the
+  NumPy documentation](https://www.numpy.org/devdocs/).
+- Producing a roadmap or list of work items for engaging the community in
+  further documentation work (we have a lot of contributor, channeling their
+  energy by setting the right goals and priorities could have a lot of impact).
+  This may include making explicit what different kinds of users need to see.
+- Rewriting a section of the User Guide (that can then serve as a template for
+  other/new sections).
+- Adding non-textual images or graphics to enhance the textual explanations
+- Updating out-of-date references and refactoring content to latest best
+  practices
+- Integrate the documentation more cleanly with the growing body of on-line
```

Comment:
nitpick: Integrating instead of Integrate

Reply:
ok